File: spring-java/src/main/java/com/jmpark/springjava/entity/ReservationEntity.java
package com.jmpark.springjava.entity;

import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.RequiredArgsConstructor;
import lombok.ToString;

import javax.persistence.Entity;
import javax.persistence.Id;
import java.time.LocalDateTime;

@Entity
@RequiredArgsConstructor
@Getter
@ToString
@EqualsAndHashCode(callSuper = false)
public class ReservationEntity extends BaseTimeEntity {
    @Id
    String id = "";

    String ticketId = null;
    LocalDateTime startAt = null;
    LocalDateTime endAt = null;
}

File: spring-kotlin/src/main/kotlin/com/jmpark/springkotlin/entity/ReservationEntity.kt
package com.jmpark.springkotlin.entity

import java.time.LocalDateTime
import javax.persistence.Entity
import javax.persistence.Id

@Entity
data class ReservationEntity (
    @Id
    val id : String = "",

    var ticketId: String? = null,
    var startAt: LocalDateTime? = null,
    var endAt: LocalDateTime? = null
) : BaseTimeEntity()